modules/test/playwright/tests/account-admin-web/accountPermissions.spec.ts

```diff
@@ -311,3 +315,118 @@ test.describe('Test for Organization Account visibility depending on Permissions
 		await expect(accountsPage.newButton).toHaveCount(0);
 	});
 });
+
+test.describe('This test asserts that an Account Admin can add and unassign an organization to an account', () => {
+	test('LPD-30009 Account admin can unassign organization to account', async ({
+		accountManagementWidgetPage,
+		accountOrganizationsPage,
+		accountsPage,
+		apiHelpers,
+		page,
+	}) => {
+		page.on('dialog', (dialog) => dialog.accept());
+
+		const userName = 'demo.unprivileged';
+		const accountAdministrator = 'Account Administrator';
+
+		await page.goto('/search');
```

Comment:
Here I would like to do something like this (let me know if can be done)
1) create a new site (so at the end we can delete the site without having to remove...)
2) create a new page with the widget inside (have a look in the `commerceChannelDefaults.spec.ts` the block `const layout = await apiHelpers.headlessDelivery.createSitePage(...`) - in this way we don't have to add manually the widget to the page --- warning: it needs the feature flag 'LPS-178052'